dev-clean/grammar/Makefile

```diff
@@ -2,6 +2,6 @@ all:
 	./generate.sh
 
 clean:
-	rm *.java
-	rm *.interp
-	rm *.tokens
\ No newline at end of file
+	rm -rf *.java
```

Comment:
`rm -f` seems enough (`-r` for directory).

Reply:
When there is no `*.java` files, running `make clean` would give an error.

Reply:
`-r` seems to be not necessary, `rm -f` won't give errors if there is no such files.

Reply:
Right... I just realized it after clicking on "comment," that's why I deleted that comment.